src/main/java/stsjorbsmod/cards/cull/Strife.java

```diff
@@ -0,0 +1,70 @@
+package stsjorbsmod.cards.cull;
+
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInDiscardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.StrengthPower;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+
+public class Strife extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(Strife.class);
+
+    private static final CardRarity RARITY = CardRarity.COMMON;
+    private static final CardTarget TARGET = CardTarget.SELF;
+    private static final CardType TYPE = CardType.SKILL;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private static final int COST = 1;
+    private static final int STRENGTH_PER_CURSE = 1;
+    private static final int UPGRADE_STRENGTH_PER_CURSE = 1;
+    private static final int CURSE_AMOUNT = 1;
+
+    public Strife() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        magicNumber = baseMagicNumber = STRENGTH_PER_CURSE;
+    }
+
+    private void calculateCurseAmount() {
```

Comment:
Consider doing this as a `calculateBonusMagicNumber` and adding the amount of strength as a dynamic card description.